### PR TITLE
Fix/uno 579/item error message hidden

### PR DESCRIPTION
**Pull request by AndreyNegrov**
**Related to task:**  https://oat-sa.atlassian.net/browse/UNO-579
**Description:** Item error message hidden
**How to test:** 
- create a item with hotspot
- adding the image in "answer"field
- Sett the cursor in question field 

**Expected result:** the warning message should not overlap with other layout elements

image

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by zagovorichev — CHANGES_REQUESTED** [review]
z-index should be fixed to not cover other popup windows

**Comment by AndreyNegrov** [comment]
> z-index should be fixed to not cover other popup windows

fixed

File: views/js/qtiCreator/tpl/notifications/invalidInfoBox.tpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="feedback-warning popup" data-widget-component="{{serial}}">
+<div class="feedback-warning-popup" data-widget-component="{{serial}}">
```
Comment (zagovorichev) [inline views/js/qtiCreator/tpl/notifications/invalidInfoBox.tpl:1]: not sure that this a good idea: this box could be used by other interactions or anything in the qtiCreator context, could you do not change classes, you could access this.$messageBox and add there your own class declaration

**Review by zagovorichev — CHANGES_REQUESTED** [review]

File: views/js/qtiCreator/tpl/notifications/invalidInfoBox.tpl (hunk 1 above)
Comment (zagovorichev) [reply views/js/qtiCreator/tpl/notifications/invalidInfoBox.tpl:1]: understood, you changed all states component, I will check it again

**Comment by zagovorichev** [comment]
I had a conversation with UI/UX (Sam) and changed this solution little bit, in this PR: https://github.com/oat-sa/extension-tao-itemqti/pull/1536

**Pull request closed without merging**